### PR TITLE
Implement my-list as advanced type on collection

### PR DESCRIPTION
**Pull request by fredrikved**
Implement my-list as advanced type on collection.

Allows for pages to be built with mylist as a section instead of a page in itself.

#325

### CODE REVIEW

**Comment by mulundapm** [comment]
Hi @fredrikved, are you able to link this to the issue?

**Comment by KillerX** [comment]
Note: I think if you want it to auto link you have to write 

Fixes <ISSUE>

File: backend/graph/api/collection-resolver.go [hunk 1]
```diff
@@ -84,24 +106,42 @@ func sectionCollectionEntryResolver(
...
+			return i.Type == "episode"
+		}), func(i *common.UserCollectionEntry, _ int) uuid.UUID {
+			return i.ItemID
+		})
+		ids, err := ls.EpisodeIDFromUuidLoader.GetMany(ctx, uuids)
+		if err != nil {
+			return nil, err
+		}
+
+		entries = filterWithIds(col, entries, ids)
```
Comment (KillerX) [inline backend/graph/api/collection-resolver.go:144]: For consideration: Personally I feel that it is not very readable where this begins and ends. I would package the block (or major parts) as a function and call it in the case.

**Review by KillerX — DISMISSED** [review]

File: backend/graph/api/collection-resolver.go (hunk 1 above)
Comment (fredrikved) [reply backend/graph/api/collection-resolver.go]: Agreed.

**Review by KillerX — APPROVED** [review]

**Pull request merged**